vllm/entrypoints/openai/api_server.py

```diff
@@ -759,6 +802,11 @@ async def init_app_state(
         state.openai_serving_models,
         request_logger=request_logger
     ) if model_config.task == "score" else None
+    state.jinaai_serving_reranking = JinaAIServingRerank(
+        engine_client,
+        model_config,
+        state.openai_serving_models,
+        request_logger=request_logger)
```

Comment:
I think we should only make it in the rerank task case, like above
```suggestion
    state.jinaai_serving_reranking = JinaAIServingRerank(
        engine_client,
        model_config,
        state.openai_serving_models,
        request_logger=request_logger
    ) if model_config.task == "rerank" else None
```

Reply:
Should use "score" task instead of "rerank" task (which doesn't exist)

Reply:
af83c253